README.md

```diff
@@ -1,11 +1,13 @@
-<h> A/B Testing/h>
+# A/B Testing
 
 <ins>**Overview**</ins>
+
 Also known as split testing, is a method of comparing two versions of a webpage or a feature of a user interface
 to determine which one performs better. It involves showing the two variants (A and B) to similar visitors at
 the same time to see the version that gives a better conversion rate.
 
 <ins>**Data Structure**</>
+
```

Comment:
Link to medallion architecture. Then you don't have to detail how the schema works. We want to be short and effective. Links to reliable sources is a great way for other people to read up more if they want to.